guides/advanced_guides/snapshots_and_dumps.md

```diff
@@ -0,0 +1,101 @@
+
+If you do not want MeiliSearch to throw an error when there is no snapshot at the given path you can add the following flag: `--ignore-missing-snapshot`. MeiliSearch will then continue its process and not import any snapshot.
+
+When starting from a snapshot, chances are that you already have an existing database. **For security reasons, a database is never overwritten**. To load a snapshot when an existing database is present, you will have to delete it manually. By default, the database is in `data.ms` folder ([unless you changed the path](/guides/advanced_guides/configuration.md#database-path)), which is located in the same folder as your MeiliSearch binary.
+The simplest way to delete your database is the use `rm -rf data.ms`. After which you can start your MeiliSearch with the path to the snapshot you want to load.
+
+[[More about snapshots flags and env variables]](/guides/advanced_guides/configuration.md#schedule-snapshot-creation)
+
+### Use Cases
+
+**Snapshots are safeguards in case of problems**. If your MeiliSearch instance encounters a problem or if you have done a mistake while manipulating your database, restarting your instance with the latest snapshot is an easy way to recover your data.
+
+### Version Compatibilities
+
+Since a snapshot is a replica of your database, it will restore it only if your MeiliSearch runs on the version it has been created on.
+
+## Dumps
+
+Dumps are compressed files containing an export of your MeiliSearch instance. It contains all your indexes, documents and settings in a raw unprocessed manner. A dump isn't an exact copy—more like a blueprint that allows you to create an identical dataset. Dumps can be imported on the launch of MeiliSearch. MeiliSearch will start after the data has been indexed.
+
+### Dumps creation
+
+To create a dump of your dataset, you need to use the appropriate http route: `POST /dumps`. Using that route will trigger a dump creation process. Creating a dump is an asynchronous task that takes time based on the size of your dataset. A dump uid (unique identifier) is returned to help you track the process.
+
+<code-samples id="post_dump_1" />
+
+### Triggers a dump creation process.
+
+At any given moment, you can check the status of a particular dump creation process using the previously received dump uid, like so: `GET /dumps/:dump_uid/status`. Using this route, you can know whether your dump is still processing, is already done, or has encountered a problem.
+
+<code-samples id="get_dump_status_1" />
+
+After your dump creation process is done, the dump file is created and added in the dump folder. By default, this folder is `/dumps` at the root of your MeiliSearch binary, but this can be customized. Note that if your dump folder does not exist when the dump creation process is called, MeiliSearch will create it.
```

Comment:
```suggestion
After your dump creation process is done, the dump file is created and added in the dump folder. By default, this folder is `/dumps` at the root of your MeiliSearch binary, but this can be customized. Note that **if your dump folder does not already exist when the dump creation process is called, MeiliSearch will create it**.
```